estate/models/estate_property.py

```diff
@@ -0,0 +1,30 @@
+from odoo import models, fields
+from datetime import date
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(string="Title", required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, readonly=True, default=date.today() + relativedelta(months=3))
```

Comment:
```suggestion
    date_availability = fields.Date(copy=False, readonly=True, default=lambda self: date.today() + relativedelta(months=3))
```
If you don't do this, it will only be evaluated one time and not work tomorrow on a long running instance.